platform/src/main/java/org/hillview/sketches/BottomKSketch.java

```diff
@@ -0,0 +1,68 @@
+package org.hillview.sketches;
+
+import it.unimi.dsi.fastutil.longs.Long2ObjectRBTreeMap;
+import net.openhft.hashing.LongHashFunction;
+import org.hillview.dataset.api.ISketch;
+import org.hillview.table.api.ColumnAndConverterDescription;
+import org.hillview.table.api.IColumn;
+import org.hillview.table.api.IRowIterator;
+import org.hillview.table.api.ITable;
+
+import javax.annotation.Nullable;
+
+/**
+ * This sketch produces a uniformly random sample of k distinct values from a column (assuming truly
+ * random hashing). The values are sampled from the *set* of distinct values taken by that column.
+ */
+public class BottomKSketch implements ISketch<ITable, MinKSet> {
```

Comment:
I think that the name of the sketch is misleading; it should probably be UniformK